TODO/whats-new-in-vue-devtools-4-0.md

```diff
@@ -2,83 +2,83 @@
+* 字面量 `Boolean` , `Number` , `Infinity` , `-Infinity` 和 `NaN`
 * Arrays
 * Plain objects
 
-For the Arrays and Plain objects, you can add and remove items using the dedicated icons. You can also rename object keys.
+对于 Arrays 和 Plain objects，可以通过专用图标来增删项。也可以重命名对象的 key 名。
 
 <iframe width="700" height="525" src="https://www.youtube.com/embed/fx1zjvHryJ0" frameborder="0" allow="autoplay; encrypted-media" allowfullscreen></iframe>
 
-A warning will be displayed if the input isn’t valid JSON. However, some values like `undefined` or `NaN` can be typed directly for more convenience.
+如果输入的不是有效的 JSON 则会显示一条警告信息。然而，为了更方便，一些像 `undefined` 或者 `NaN` 的值是可以直接输入的。
 
-More types will be supported in future releases!
+未来的新版本会支持更多类型的！
 
-#### Quick Edit
+#### 快速编辑
 
-Some types of value can be edited in a single click with the ‘Quick Edit’ feature.
+通过 “快速编辑” 功能可以实现仅仅鼠标单击一下，就可以编辑一些类型的值了。
 
-Booleans can be toggled directly with the checkbox icon:
+布尔值可以直接通过复选框进行切换：
 
 <iframe width="700" height="525" src="https://www.youtube.com/embed/llNJapRZaHo" frameborder="0" allow="autoplay; encrypted-media" allowfullscreen></iframe>
 
-Numbers can be incremented or decremented with the plus or minus icons:
+数值可以通过加号和减号图标进行增减：
 
 <iframe width="700" height="525" src="https://www.youtube.com/embed/ZCToaOpId0w" frameborder="0" allow="autoplay; encrypted-media" allowfullscreen></iframe>
 
-You can use some keyboard modifiers to increment or decrement the value faster.
+使用键盘的修改键去进行增减会更快一些。
 
-### Open component in editor
+### 在编辑器中打开一个组件
 
-If you are using vue-loader or Nuxt in your project, you can now open the selected component in your favorite code editor (provided it is a Single-File Component).
+如果项目中使用了 vue-loader 或 Nuxt 的话，现在你就可以在你最喜欢的编辑器里打开选定的组件（只要它是单文件组件）。
 
-1. Follow this [setup guide](https://github.com/vuejs/vue-devtools/blob/master/docs/open-in-editor.md) (if you are using Nuxt, you don’t need to do anything)
-2. In the Component inspector, mouse over the component name — you should see a tooltip with the file path
-3. Click on the component name and it will open in your editor
+1. 按这份 [设置指南](https://github.com/vuejs/vue-devtools/blob/master/docs/open-in-editor.md) 操作 （如果你使用的是 Nuxt，就什么都不用做）
+2. 在组件检查器中，将鼠标移动到组件名上 —— 你会看到一个显示文件路径的提示框
+3. 单击组件名就会直接在编辑器中打开该组件了
 
 <iframe width="700" height="525" src="https://www.youtube.com/embed/XBKStgyhY18" frameborder="0" allow="autoplay; encrypted-media" allowfullscreen></iframe>
 
-### Show the original component names
+### 显示原始的组件名
 
-PR by [manico](https://github.com/manico)
+这一功能由 [manico](https://github.com/manico) 提出的 PR 实现
 
-By default, all the component names are formatted into CamelCase. You can disable this by toggling the ‘Format component names’ button in the Components tab. This settings will be remembered and it will also be applied to the Events tab.
+默认情况下，组件名都会被格式化为驼峰形式。你可以通过切换组件标签下的 "Format component names" 按钮来禁用这一功能。这一设置会被记忆并应用到 Event 标签页中。
```

Comment:
This settings will be remembered and it will also be applied to the Events tab。
译者应该翻译为 Events 标签页
(校对者注：这个地方校对者不太理解原作者的意思，应用到Events tab是？是用户对于编辑器的自定义设置里面嘛)

Reply:
这一设置会被记忆并应用到 Event 标签页中。 => 这个设置将被记住，它也将被应用到 Events 标签页中。

Reply:
@okaychen 可以看下视频，组件标签页中有 “Format component names” 按钮，事件标签页中也有这样一个按钮

Reply:
😄 ，Thanks @FateZeros